models/linkages/R/read_restart.LINKAGES.R

```diff
@@ -28,8 +28,8 @@ read_restart.LINKAGES <- function(outdir, runid, stop.time, settings, var.names
   ens <- read.output(runid = runid, 
                      outdir = file.path(outdir, runid), 
                      start.year = lubridate::year(stop.time), 
-                     end.year = lubridate::year(stop.time), 
-                     variables = var.names)  # change to just 'AGB' for plot level biomass
+                     end.year = lubridate::year(stop.time),
+                     variables = var.names,pft.name = lapply(settings$pfts,'[[','name'))  # change to just 'AGB' for plot level biomass
```

Comment:
have you checked if `pft.name = lapply(settings$pfts,'[[','name')` works? I think read.output expects a string, something like `unlist(sapply(settings$pfts, `[[`, "name"))` as you have in write.config.linkages below

just wanted to note this

Reply:
yeah, it's been working but I'll check again